compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/SearchBar.skiko.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.material3
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalWindowInfo
+import androidx.compose.ui.unit.Dp
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+internal actual fun getScreenHeight(): Dp {
+    val density = LocalDensity.current
+    val windowInfo = LocalWindowInfo.current
+    return with(density) {
+        windowInfo.containerSize.height.toDp()
+    }
+}
```

Comment:
Very minor stylistic nit, but maybe better looking without the local variables:
```
return with(LocalDensity.current) {
    LocalWindowInfo.current.containerSize.height.toDp()
}
```

Reply:
Well, it was my initial implementation. But I introduced var for better looking 😄